.env

```diff
@@ -0,0 +1,20 @@
+# MySQL environment variables
+DB_PASSWORD=123sqlbod321
+DB_DATABASE=book_store_db
+DB_USER=bodya
+DB_LOCAL_PORT=3309
+DB_DOCKER_PORT=3306
+
+# Spring Boot application environment variables
+SPRING_LOCAL_PORT=8080
+SPRING_DOCKER_PORT=8080
+
+# Debug port
+DEBUG_PORT=5005
+
+# JWT secret key
+JWT_SECRET=wefwqfdfwqfq4f3fq3af43453452ef34fewfewfe
+
+MYSQL_DATABASE=book_store_db
+MYSQL_USER=bodya
+MYSQL_PASSWORD=123sqlbod321
```

Comment:
It is better to not push env file, you can add an example of it with redacted values in the readme, but do ton push the file itself

Reply:
Not resolved